src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -75,7 +76,10 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
             {
                 if (!ex.IsCriticalException())
                 {
-                    notifications.ShowError(ex.Message);
+                    //TODO:Will need a uniform solution to HTTP exception message handling
+                    var apiException = ex as ApiValidationException;
+                    var error = apiException?.ApiError?.Errors?.FirstOrDefault();
+                    notifications.ShowError(error.Message ?? ex.Message);
```

Comment:
`error` can be null here, so it'll throw if you do `error.Message`, you need to null check there as well.